templates/conditionorc-configmap.yaml

```diff
@@ -16,6 +16,9 @@ data:
       - kind: firmwareInstall
         exclusive: true
         failoncheckpointerror: true
+    serverservice:
+      endpoint: "{{ .Values.conditionorc.env.SERVERSERVICE_ENDPOINT }}"
+      disable_oauth: "{{ .Values.conditionorc.env.SERVERSERVICE_DISABLE_OAUTH }}"
```

Comment:
I think these are suppose to be `ALLOY_SERVERSERVICE_ENDPOINT` and `ALLOY_SERVERSERVICE_DISABLE_OAUTH`, right?

Reply:
Oh, I see now. The ones for conditionorc didn't change. I should have expanded the code in GH first.